test-site/config/index.json

```diff
@@ -36,7 +36,10 @@
     },
     "KM": {
       "universalSectionTemplate": "grid-two-columns",
-      "cardType": "multilang-financial-professional-location"
+      "cardType": "multilang-financial-professional-location",
+      "mapConfig": {
+        "mapProvider": "Google"
```

Comment:
nit: Is switching this to google intentional or just for testing?

Reply:
just for testing, before this we didn't display a map on universal at all